IaC/BareMetal/README.md

```diff
@@ -4,6 +4,9 @@
 
 ## Create IMDb Cosmos DB
```

Comment:
I think we should rename BareMetal to "DevCluster" or something similar per email thread